test/tools/libtesteth/TestHelper.cpp

```diff
@@ -242,21 +243,31 @@ void replaceLLLinState(json_spirit::mObject& _o)
 	}
 }
 
-std::vector<boost::filesystem::path> getJsonFiles(boost::filesystem::path const& _dirPath, std::string const& _particularFile)
+std::vector<boost::filesystem::path> getFiles(boost::filesystem::path const& _dirPath, string const& _extentionMask, std::string const& _particularFile)
 {
 	vector<boost::filesystem::path> jsonFiles;
+	vector<string> extentions;
```

Comment:
If you want a `vector` here use it as an argument. Another alternative to `std::set`.